### ISSUE TITLE
[a11y-defect-2]: Name of issue input field's hint text should describe the field

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#63227]
## Point of contact

@joshkimux

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

## Details

Currently, the name of issue's input field has visible hint text that is not programmatically linked to the input field itself. As a result, screen reader user's tabbing through the form will not be able to hear the additional details of "you can only add an issue that you've already received a VA decision notice for."

<img width="1300" alt="Screenshot 2023-08-07 at 2 06 27 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/14154792/ac91735e-acd2-4fe5-b889-0095473c703b">

## Reproduction steps

1. Log in as user 228
2. Navigate to "add an issue" page
3. Turn on your screen reader of choice
4. Tab to the first input field "Name of issue"
5. Confirm that the hint text "You can only add an issue..." is not read out
...

## Proposed solution or next steps

If possible, we should follow [VA design system's guidance](https://design.va.gov/storybook/?path=/docs/components-va-text-input--with-hint-text) of using the hint prop which should programmatically connect the hint text to the input field.

### Acceptance criteria
- [ ] Hint text is programmatically connected to the name of issue input field
- [ ] Hint text ID's are unique

## References, articles, or WCAG support

1.
2.
3.
...

## Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by HeatherWidmont** [comment]
@jaredcunha do you have capacity to review this?

**Comment by jaredcunha** [comment]
<img width="538" alt="Screenshot 2023-09-19 at 9 19 03 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/1094951/ee385fd6-a1c5-4d33-8a5c-9d4171410a9e">

Can confirm the description is present in the accessibility tree.

**Comment by HeatherWidmont** [comment]
Thank you so much @jaredcunha!! @Mottie does that mean this ticket is ready to close?

**Comment by Mottie** [comment]
Yes, let's close this!
